packages/rum-core/src/common/patching/fetch-patch.js

```diff
@@ -94,13 +94,18 @@ function patchFetch(callback) {
       promise.then(
         function(response) {
           resolve(response)
-          task.data.response = response
-          invokeTask(task)
+          // invokeTask in the next execution cycle to let the promise resolution complete
+          setTimeout(() => {
```

Comment:
I am not sure if this would be the right approach. Any reason why you went with setTimeout instead of Promise.resolve?

Reply:
After reading through a bit, promise.resolve is more appropriate here than setTimeout and promise.resolve also guarantees that our next function would be in the order after the previous resolution happened since both of the resolutions will be part of microTask queue.

Reply:
Yeah, It was my temp solution will use promise.resolve.